tutorials/advanced_features/Simulating with SPAM errors.ipynb

```diff
@@ -0,0 +1,501 @@
+{
```

Comment:
<p>Here you just copy pasted what I suggested...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/460/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>